### PR TITLE
Improve admin dashboards

### LINKED ISSUE #81: Improve admin dashboard tables
- [x] Add clear all filters button
- [x] Improve filter indicator look and feel

### PR DESCRIPTION
**Pull request by AdrianKokot**
Closes #81

### CODE REVIEW

File: frontend/modules/admin/src/lib/lib.routes.ts [hunk 1]
```diff
@@ -39,13 +39,13 @@ export const adminRoutes: Route[] = [
       {
         title: translatedTitle('KEYWORDS'),
         path: 'keywords',
-        canMatch: [AuthorizedOfTypeGuard(ExtendedAccountTypeEnum.ADMIN)],
+        canActivate: [AuthorizedOfTypeGuard(ExtendedAccountTypeEnum.ADMIN)],
```
Comment (mudziok) [inline frontend/modules/admin/src/lib/lib.routes.ts:42]: non-blocking:
Why was canMatch replaced with canActivate?

File: frontend/modules/shared/security/src/lib/permission.service.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+})
+export class PermissionService {
+  private readonly _user$ = inject(AuthService).user$.pipe(share());
+
+  public canMatch(requiredRole: ExtendedAccountType): Observable<boolean> {
+    return this._user$.pipe(
+      first(),
+      map((user) => isUserOfType(user, requiredRole)),
+      tap((canMatch) =>
+        console.debug('PermissionService@canMatch', requiredRole, canMatch),
```
Comment (mudziok) [inline frontend/modules/shared/security/src/lib/permission.service.ts:18]: Probably leftover log?

File: frontend/modules/admin/src/lib/admin.style.less [hunk 3]
```diff
@@ -117,9 +117,10 @@
   color: inherit;
 }
 
-reunice-table-filter._applied-filter .t-icon {
-  color: var(--tui-primary);
-}
+//reunice-table-filter._applied-filter .t-icon {
+//  background-color: var(--tui-info-bg);
```
Comment (mudziok) [inline frontend/modules/admin/src/lib/admin.style.less:121]: Leftover styles

**Review by mudziok — APPROVED** [review]
looks cool especially to see tests
approved as none of my comments are blocking tbh

File: frontend/modules/admin/src/lib/lib.routes.ts (hunk 1 above)
Comment (AdrianKokot) [reply frontend/modules/admin/src/lib/lib.routes.ts:42]: Leftover after debugging why for example `/admin` url was always redirected to `/` even if user was logged in, I'll rollback this change

**Pull request merged**